plugins/modules/site_intent.py

```diff
@@ -90,18 +88,27 @@
               name:
                 description: Name of the floor (eg floor-1).
                 type: str
-              parentName:
-                description: Complete Parent name of the floor to be created(eg Global/USA/San Francisco/BGL_18).
+              parent_name:
+                description: Complete parent name of the floor to be created(eg Global/USA/San Francisco/BGL_18).
                 type: str
               rf_model:
-                description: Type of floor. Allowed values are 'Cubes And Walled Offices',
-                  'Drywall Office Only', 'Indoor High Ceiling', 'Outdoor Open Space'.
+                description: Type of floor (allowed values are 'Cubes And Walled Offices', 'Drywall Office Only', 'Indoor High Ceiling',
+                    'Outdoor Open Space'). It refers to the Radio Frequency (RF) model of the floor. It is essential in wireless
+                    networking to simulate and optimize radio signal propagation and coverage within a physical space.
+                    Cubes And Walled Offices - This RF model typically represents indoor areas with cubicles or walled offices, where
+                        radio signals may experience attenuation due to walls and obstacles.
+                    Drywall Office Only - This RF model indicates an environment with drywall partitions, commonly found in office spaces,
+                        which may have moderate signal attenuation.
+                    Indoor High Ceiling - This RF model is suitable for indoor spaces with high ceilings, such as auditoriums or atriums,
+                        where signal propagation may differ due to the height of the ceiling.
+                    Outdoor Open Space - This RF model is used for outdoor areas with open spaces, where signal propagation is less obstructed
+                        and may follow different patterns compared to indoor environments.
                 type: str
               width:
                 description: Width of the floor units is ft. (eg 100).
                 type: int
               floor_number:
-                description: Floor number in the building/site (eg 5).once created, it can't be modified.
+                description: Floor number in the building/site (eg 5) can be given only while creating the floor site.
```

Comment:
description: Floor number within the building/site (e.g., 5). This value can only be specified during the creation of the floor and cannot be modified afterward.

Reply:
updated.